sql/parser2/eval_test.go

```diff
@@ -134,7 +132,6 @@ func TestEvalExpr(t *testing.T) {
 		// Comparisons against NULL result in NULL, except for the null-safe equal.
```

Comment:
looks like there is no more null-safe equal, so this comment (and maybe others?) should be updated